app/views/RoomMembersView/index.js

```diff
@@ -139,6 +139,7 @@ class RoomMembersView extends React.Component {
 		let membersFiltered = [];
 
 		if (members && members.length > 0 && text) {
+			text = text.trim();
```

Comment:
It should be done before the if condition right above, otherwise an empty space is going to be a valid search text.

Reply:
Moved the code upwards. It should work now!